client/app/visualizations/chart/plotly.js

```diff
@@ -514,9 +517,11 @@ const CustomPlotlyChart = (clientConfig) => {
           });
         });
       };
-      scope.$watch('options.customCode', () => {
+      scope.$watch('[options.customCode, options.autoRedraw]', () => {
```

Comment:
Why do we need to watch `autoRedraw`? If it's disabled, on next refresh it won't redraw. If it's enabled, still we have nothing to do.